tests/streams/test_dash.py

```diff
@@ -208,6 +208,26 @@ def test_parse_manifest_drm(self, mpdClass):
                           self.session, self.test_url)
         mpdClass.assert_called_with(ANY, base_url="http://test.bar", url="http://test.bar/foo.mpd")
 
+    @patch('streamlink.stream.dash.MPD')
```

Comment:
I don't think this test is quite valid, it doesn't test that the correct thing is passed to the MPD class. I think parsing one of the included resources might be a better test.

```python
    def test_parse_manifest_string(self):
        with text("dash/test_9.mpd") as mpd_txt:
            test_manifest = mpd_txt.read()

        streams = DASHStream.parse_manifest(self.session, test_manifest)

        self.assertSequenceEqual(
            sorted(list(streams.keys())),
            sorted(["2500k"])
        )
```

Reply:
fixed in 6aea0f4